### PR TITLE
PA128

### PR DESCRIPTION
**Pull request by dorschw**
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10040

### CODE REVIEW

File: demisto_sdk/commands/validate/validators/PA_validators/PA128_validate_pack_files.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    ValidationResult,
+)
+
+ContentTypes = Pack
+
+
+class PackFilesValidator(BaseValidator[ContentTypes]):
+    error_code = "PA128"
+    description = "Checks for required pack files"
+    rationale = "These files are required by the platform."
```
Comment (yuvalbenshalom) [inline demisto_sdk/commands/validate/validators/PA_validators/PA128_validate_pack_files.py:23]: not sure the rational is accurate. 
.secrets_ignore and .pack-ignore are required in order to maintain content repo, not the platform.

File: demisto_sdk/commands/validate/validators/PA_validators/PA128_validate_pack_files.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    error_message = "Packs require a .secrets_ignore, .pack-ignore and README"
+    fix_message = "Created required files, empty."
+    related_field = ""  # TODO?
```
Comment (yuvalbenshalom) [inline demisto_sdk/commands/validate/validators/PA_validators/PA128_validate_pack_files.py:26]: TODO?

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
see my comments

**Comment by yuvalbenshalom** [comment]
also, changing is needed?

File: demisto_sdk/commands/validate/validators/PA_validators/PA128_validate_pack_files.py (hunk 2 above)
Comment (dorschw) [reply demisto_sdk/commands/validate/validators/PA_validators/PA128_validate_pack_files.py:26]:
```suggestion
    related_field = "secrets-ignore,pack-ignore,readme"
```

File: demisto_sdk/commands/validate/validators/PA_validators/PA128_validate_pack_files.py (hunk 1 above)
Comment (dorschw) [inline demisto_sdk/commands/validate/validators/PA_validators/PA128_validate_pack_files.py:23]:
```suggestion
    rationale = "These files are used as standard in the demisto/content repo."
```

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**